apps/src/common-styles.module.scss

```diff
@@ -20,6 +20,12 @@
   &Unfocused {
     background-color: color.$lighter_purple;
     color: color.$dark_charcoal;
+
+    button {
+      span {
+        color: color.$dark_charcoal;
+      }
+    }
```

Comment:
Just confirming, do we want to apply this to all `button span`s, not just `PaneButton`? also should we copy this into commonStyles.js (based on the comment at the top of the file)?

Reply:
We're applying this to all the buttons that are inside of purpleHeader class right now. And the only buttons inside that purpleHeader is paneButtons
<img width="322" alt="image" src="https://user-images.githubusercontent.com/22244040/222268481-babcc65a-2b65-4767-9f25-368a534973b3.png">

As far as I remember I've asked about commonStyles.js and the answer was not to use it and get rid of it where possible